src/app/parcels/ParcelsPage.tsx

```diff
@@ -358,6 +357,19 @@ const ParcelsPage: React.FC<{}> = () => {
 
     const [sortState, setSortState] = useState<SortState<ParcelsTableRow>>({ sortEnabled: false });
 
+    const [perPage, setPerPage] = useState(10);
```

Comment:
shall we rename this to parcelCountPerPage while we're at it?

Reply:
![image](https://github.com/NorwoodAndBrixtonFoodbank/nbf-website/assets/138508884/ce0c2860-01f4-4ccf-ac82-afd259158ac0)